server/@types/make-recall-decision-api/models/PpudDetailsResponse.ts

```diff
@@ -27,7 +27,7 @@ export type PpudDetailsSentence = {
   mappaLevel: string,
   licenceExpiryDate: string,
   offence: PpudDetailsOffence,
-  releases: PpudDetailsRelease[],
+  releaseDate: string,
```

Comment:
We're not setting strictNullChecks=true, so evil things like
`var x: string = null`
are valid in type TypeScript project at the moment. I've not changed it since the whole type is passed nullable strings from the API but deals with them all as strings.